instrumentation/opentelemetry_redix/lib/opentelemetry_redix.ex

```diff
@@ -27,19 +27,30 @@ defmodule OpentelemetryRedix do
 
   @doc """
   Initializes and configures the telemetry handlers.
+
+  You may also supply the following options:
+
+    * `db_statement` - `:enabled`, `:disabled` or a single arity function. If
+      `:enabled`, the DB statement is sanitized of sensitive values, such as
+      AUTH crendentials, but otherwise logged. If `:disabled`, the statement is
+      not logged. If you pass in a single arity function, the list of commands
```

Comment:
My suggestion is to have a type for the function to point to and document better. For example:

```elixir
@type db_statement_fun() :: (Redix.command() -> String.t())
```